src/Semoda/Semoda/ViewModels/DashboardPageViewModel.cs

```diff
@@ -13,6 +18,25 @@ public class DashboardPageViewModel : ViewModelBase
         /// </summary>
         public DashboardPageViewModel() : base(true)
         {
+            IPerformanceDataService dataService = ServiceProvider.GetRequiredService<IPerformanceDataService>();
+
+            _ = dataService.RegisterAsync(HandleCPUUtilization, Models.PerformanceDataType.TotalCPUUtilization).ContinueWith((t) =>
+            {
+                _ = dataService.StartAsync();
+                _ = dataService.RegisterAsync(HandleRAMAvailable, Models.PerformanceDataType.TotalRAMAvailable);
+            });
+        }
+
+        private void HandleCPUUtilization(object? sender, PerformanceDataEventArgs args)
+        {
+            if (args.PerformanceDataType == Models.PerformanceDataType.TotalCPUUtilization)
```

Comment:
seems inelegant to have to do this if we're already registering 2 different methods to catch the 2 different events, but i don't know enough about the language and framework to tell if there's a better way.

Reply:
We need to register twice, because every performance counter handles a specific parameter. But indeed it is possible to merge both HandleMethods to one method. 
The check which kind of performance data is collected needs to be done, because every performance data runs through the same event bus. Otherwise we would need to have a single EventHandler for every possible future data type.